### PR TITLE
add CosmWasm IDE documentation

### PR DESCRIPTION
**Pull request by ducphamle2**

### CODE REVIEW

**Review by orkunkl — COMMENTED** [review]
Frontend section is reserved for cosmjs apps. 
Looks like we can reorg the documentation a bit I will do that part later.
Is it possible to give an ide tutorial in this docs?

**Comment by ducphamle2** [comment]
> Frontend section is reserved for cosmjs apps. Looks like we can reorg the documentation a bit I will do that part later. Is it possible to give an ide tutorial in this docs?

Alright please help reorg the documentation since I'm not sure where to put the CosmWasm IDE section.

And yes I will add a tutorial on how to use it with Gitpod & Keplr.

**Comment by orkunkl** [comment]
I checked out the docs, you can actually include this cosmwasm IDE to [setting up your ide docs](https://docs.cosmwasm.com/docs/1.0/getting-started/installation#setting-up-your-ide). And have the tutorial hosted under https://docs.cosmwasm.com/tutorials and IDE github repo. Also I think tutorial doc will make a good medium article.

**Comment by ducphamle2** [comment]
I think the tutorial doc should not be written as a medium article, as the content will focus on CosmWasm smart contract developers. Meanwhile, a medium article shall target general audiences who don't know much about technology and coding.

**Comment by ducphamle2** [comment]
We can probably record a video tutorial though, which can be uploaded onto Youtube and embed the vid's link into the doc page

File: docs/02-getting-started/02-installation.md [hunk 1]
```diff
@@ -131,3 +131,19 @@ There are many more editors out there and some have varying degrees of rust supp
 unless you have a strong preference to another editor (e.g. Sublime, Emacs, Vim) trying one of the two above is
 recommended, especially if you are new to rust. Once you are confident in the language, you can always use another
 editor and customize it to your liking.
+
+[CosmWasm IDE](https://github.com/oraichain/cw-vscode)
+
+CosmWasm IDE is a tool that simplifies the CosmWasm smart contract development & deployment processes. It integrates with Gitpod & Keplr to create a simple yet powerful environment to build, deploy, and interact with the CosmWasm smart contracts through default and custom networks using CosmWasm. The tool is currently maintained by Oraichain & CosmWasm.
```
Comment (orkunkl) [inline docs/02-getting-started/02-installation.md]: Can you also mention `with gitpod you can develop on the browser and it is well maintained.`

**Review by orkunkl — APPROVED** [review]

**Pull request merged**